docs/reports/peer_testing_2/usability_tasks.md

```diff
@@ -106,12 +121,13 @@ and will be adding more avatars in the future.
 
 Read the README, and then identify:
 
-1. How many parameters are sent to the API?
-2. What are the parameters that are sent to the API?
+1. The instructions in the README are clear
+2. Which parameters are sent to the API?
 
 ### Developer Scenario 2: Character Assets
 
-1. Famliarize yourself with the program structure.
-2. Identify where the character image assets are saved.
-3. Tell the examiner about the character image file naming scheme and how the names relate to the images
-4. If you were going to add a new character, how would you do it?
+1. Identify where the character image assets are saved.
+2. Does the naming scheme of the character image assets make senese?
+3. What does the naming scheme tell you about the images?
+4. If an artist provided you with the Adobe Illustrator file for a new character, I could add the new character's assets.
+5. Using the README, create and add the image assets for Boy. Did you succeed?
```

Comment:
This might not be a feasible one, as creation will require them to have illustrator installed

Reply:
Depending on the schedule, we could just use my laptop for this. But you're right. I didn't think of that. Woopies